### PR TITLE
fix obj vertex//normal face format

### PR DESCRIPTION
**Pull request by rdeits**
The order of `if` statements in `obj.jl` was backwards, which caused faces of the form `f 1//1 2//2 3//3` to be split incorrectly and broke loading of those meshes. This PR fixes that comparison and adds a test of that kind of mesh.

### CODE REVIEW

File: src/io/obj.jl [hunk 1]
```diff
@@ -34,10 +34,10 @@ function load{MT <: AbstractMesh}(io::Stream{format"OBJ"}, MeshType::Type{MT} =
                     error("Unknown UVW coordinate: $lines")
                 end
             elseif "f" == command #mesh always has faces
-                if any(x->contains(x, "/"), lines)
```
Comment (rdeits) [inline src/io/obj.jl]: This was always true, even for meshes that contained `//` instead of just `/`

**Comment by rdeits** [comment]
cc @tkoolen

**Comment by rdeits** [comment]
Ok, @SimonDanisch I think this is working on 0.6 (and failing due to unrelated FileIO issues on 0.7)

**Comment by SimonDanisch** [comment]
Thanks for this! :)

**Pull request merged**